deploy/go-carbon.spec

```diff
@@ -0,0 +1,78 @@
+%define debug_package %{nil}
+
+Name:	    go-carbon
+Version:	%{version}
+Release:	%{release}%{?dist}
+Summary:	Carbon server for graphite
+
+Group:		Development/Tools
+License:	MIT License
+URL:		https://github.com/lomik/go-carbon
+Source0:	%{name}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}
+
+%if systemd_requires
+%systemd_requires
+%else
+Requires:  /etc/init.d/functions
```

Comment:
Can we change this to "initscripts" this breaks yum dependencies  in amazon linux.

```
Error: Package: go-carbon-0.9.1-1.el6.x86_64 (@timsrepo-rpm-repo)
           Requires: /etc/init.d/functions
           Removing: initscripts-9.03.49-1.34.amzn1.x86_64 (installed)
               Not found
           Updated By: initscripts-9.03.49-1.35.amzn1.x86_64 (amzn-updates)
               Not found
yum whatprovides /etc/init.d/functions
Loaded plugins: priorities, s3iam, update-motd, upgrade-helper
initscripts-9.03.49-1.34.amzn1.x86_64 : The inittab file and the /etc/init.d scripts
Repo        : installed
Matched from:
Filename    : /etc/init.d/functions
```